src/include/command.h

```diff
@@ -25,6 +25,8 @@
 
 #include "target.h"
 
+extern bool do_probe;
```

Comment:
Please name this more along the lines of `no_skip_cortexm_probe` - we know this is longer but it makes it clearer what this actually does.